packages/@aws-cdk/integ-tests-alpha/lib/assertions/sdk.ts

```diff
@@ -50,19 +50,6 @@ export interface AwsApiCallProps extends AwsApiCallOptions { }
 export class AwsApiCall extends ApiCallBase {
   public readonly provider: AssertionsProvider;
 
-  /**
-   * access the AssertionsProvider for the waiter state machine.
-   * This can be used to add additional IAM policies
-   * the the provider role policy
-   *
-   * @example
-   * declare const apiCall: AwsApiCall;
-   * apiCall.waiterProvider?.addToRolePolicy({
-   *   Effect: 'Allow',
-   *   Action: ['s3:GetObject'],
-   *   Resource: ['*'],
-   * });
-   */
   public waiterProvider?: AssertionsProvider;
```

Comment:
Looks like the build is failing here. Can't remove this docstring entirely since its a public API. Why was it removed in the first place?

Reply:
I see, I added it anyway.

I've moved the same explanation to `IApiCall`, and at few lines up `public readonly provider: AssertionsProvider;` had no docstring, so I removed this too. Why is it good that there is no docstring there (`provider`)?

Reply:
I see, that's because it's readonly.